micro-rdk/src/native/certificate.rs

```diff
@@ -26,7 +26,7 @@ impl WebRtcCertificate {
         let cert = rcgen::Certificate::from_params(param).unwrap();
         let cert_der = cert.serialize_der().unwrap();
 
-        let fp_hashed = Sha256::new_with_prefix(&cert_der)
+        let fp_hashed = Sha256::new_with_prefix(cert_der.clone())
```

Comment:
this one I'm not sure about. might be better to ignore the warning instead of cloning.

Reply:
Can you try `.into` here?